# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/webrender_bindings/RendererScreenshotGrabber.h
# Commit: 971767c6ed2b
# Full Hash: 971767c6ed2b2201e172105046ecc61942e03a7e
# Author: sotaro <sotaro.ikeda.g@gmail.com>
# Date: 2020-01-24 21:46:30
# Regressor Bug: 1602643
# File Overlap Count: 1
# Description:
#   Bug 1602643 - Disable WebRender compositor dinamically for async screenshot r=gw
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D59658
# ==============================================================================

diff -r 436df7dc06c6 -r 971767c6ed2b gfx/webrender_bindings/RendererScreenshotGrabber.h
--- a/gfx/webrender_bindings/RendererScreenshotGrabber.h	Fri Jan 24 01:04:09 2020 +0000
+++ b/gfx/webrender_bindings/RendererScreenshotGrabber.h	Thu Jan 23 23:52:05 2020 +0000
@@ -32,14 +32,14 @@
    * The captured screenshot will not be mapped until the second call to
    * |MaybeProcessQueue| after this call to |MaybeGrabScreenshot|.
    */
-  void MaybeGrabScreenshot(Renderer* aRenderer,
+  void MaybeGrabScreenshot(RendererOGL* aRendererOGL,
                            const gfx::IntSize& aWindowSize);
 
   /**
    * Process the screenshots pending in the queue if we are profiling and
    * screenshots are enabled.
    */
-  void MaybeProcessQueue(Renderer* aRenderer);
+  void MaybeProcessQueue(RendererOGL* aRenderer);
 
  private:
   /**